charts/datadog/values.yaml

```diff
@@ -377,7 +376,7 @@ clusterAgent:
 
     # clusterAgent.metricsProvider.useDatadogMetrics -- Enable usage of DatadogMetric CRD to autoscale on arbitrary Datadog queries
     ## NOTE: You need to install the `DatadogMetric` CRD before
-    useDatadogMetrics: false
+    useDatadogMetrics: true
```

Comment:
I think it is ok, but we need to document that we only supported kubernetes > 1.10 (to support crds)

Reply:
So we agree to drop Kube < 1.10 (I think ATM we support 1.8?)